ci-operator/step-registry/openshift/e2e/openstack/minimal/openshift-e2e-openstack-minimal-workflow.yaml

```diff
@@ -6,9 +6,6 @@ workflow:
     post:
     - chain: ipi-openstack-post
     env:
-      OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
```

Comment:
Perhaps we could leave those, so that different flavors could be tested (some large flavor for SNO / compact jobs)?

Reply:
The current workflows don't need the values, because the names are selected automatically. However, if you do set these values in a workflow, they will be used (see the other comment).